wire-ios-mono.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -10,6 +10,24 @@
           "version": "1.3.0"
         }
       },
+      {
+        "package": "AsyncAlgorithms",
```

Comment:
do we need that @typfel ?

Reply:
No, I thought I removed it again but I guess missed something.

Reply:
ok i'll remove